src/main/paradox/docs/tutorial/knowledge-graph/thinking-in-graph.md

```diff
@@ -39,26 +39,24 @@ and he has a property **was born on** for this the value is **14 July 1990**.
 the data and derive new information from explicitly asserted data. This inference can make information easier to
 discover.
 
- In the example below, we have **Bob**, he has parents (using the relation called **parent**) and we have asserted that
-his father also has parents. Having define this, we can now define a rule that instructs the inference engine to create
-new relations **grand parent** whenever we find the **parent** of a **parent**. This enables us to query the graph
-and directly get **Bob**'s grand parents without having to navigate through his parents first.
+ In the example below, we have **Toy Story**, it has characters (using the relation called **Character**) and we have asserted the cast of characters. Having define this, we can now define a rule that instructs the inference engine to create
+new relations **Voice Cast** whenever we find the **Cast** of a **Character**. This enables us to query the graph
+and directly get **Toy Story**'s voices cast without having to navigate through its cast first.
 
- ![Graph Structure](./assets/kg-smart.png)
+ ![Graph Structure](./assets/kg_inferred-relation0.png)
```

Comment:
the image `./assets/kg_inferred-relation0.png` is not present in the PR, I'm not sure if it was already present before or you forgot to add it to the commit?